angular.json

```diff
@@ -74,7 +62,6 @@
               "optimization": true,
               "outputHashing": "all",
               "sourceMap": false,
-              "extractCss": true,
```

Comment:
Why was this removed?

Reply:
I this [version ](https://v11.angular.io/cli/build#:~:text=Deprecated%3A%20Deprecated%20since,instead%20of%20%27.js%27.)be come deprecated and in version 13 removed